### DISCUSSION TITLE
Mac Compatability

### DISCUSSION THREAD

**Discussion by cparish312** [apirrone/Memento#81] [General]
Hey all, I'm planning on making Memento Mac compatible. Curious if anyone else is working on this or has any ideas that could be helpful! I was actually building out a python app to do exactly this before I discovered Memento, and one major thing I discovered is the mac build in OCR (I used https://github.com/straussmaximilian/ocrmac) is significantly better than tesseract. I plan on adding that as an OCR option if running on Mac.

**Comment by apirrone** [comment]
Hi @Connorpar, that's amazing! Keep me updated on your progress, and do not hesitate to ask if anything is unclear :)
